examples/rails/Gemfile

```diff
@@ -3,7 +3,8 @@ source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.6'
-# Use sqlite3 as the database for Active Record
+# Use postgres as the database for Active Record
+gem 'pg'
 gem 'sqlite3'
```

Comment:
Believe @nzoschke added this back as some of the tools were complaining. Perhaps he can explain why better than me.